tools/resourcedocsgen/cmd/docs/registry.go

```diff
@@ -35,6 +36,7 @@ import (
 	pschema "github.com/pulumi/pulumi/pkg/v3/codegen/schema"
 	"github.com/pulumi/pulumi/sdk/v3/go/common/util/contract"
 	"github.com/pulumi/registry/tools/resourcedocsgen/pkg"
+	concpool "github.com/sourcegraph/conc/pool"
```

Comment:
Any chance this can stick to stdlib? Perhaps no big deal.

Reply:
This is from #5917. The only changes unique to this PR are in scripts files. I'll rebase before merging to make it clearer.